playwright.config.ts

```diff
@@ -26,7 +26,7 @@ export default defineConfig({
       { open: 'never', outputFolder: path.join(testDir, 'playwright-report') },
     ],
   ],
-  use: { trace: 'on-first-retry' },
+  use: { trace: 'retain-on-failure' },
```

Comment:
Or do we want this?
```suggestion
  use: { trace: 'on' },
```

Reply:
No, checked docs. One you added is best one.

Reply:
I don't mind having this. Are we going to use the traces from successful runs as well?

Reply:
No. I just checked docs. retain-on-failure is best.